subsys/bluetooth/host/keys.h

```diff
@@ -102,3 +105,5 @@ struct bt_keys_link_key *bt_keys_get_link_key(const bt_addr_t *addr);
 struct bt_keys_link_key *bt_keys_find_link_key(const bt_addr_t *addr);
 void bt_keys_link_key_clear(struct bt_keys_link_key *link_key);
 void bt_keys_link_key_clear_addr(const bt_addr_t *addr);
+
+void bt_keys_notify_usage(u8_t id, const bt_addr_le_t *addr);
```

Comment:
missing API documentation

Reply:
Where should it be?

Reply:
@nvlsianpu this is a private API, so it's not required, but if you want you can have an informal code comment above the prototype clarifying usage & purpose.

Reply:
^^ would be enough to have just clarification comment above on what it does.

Reply:
fixed